### ISSUE TITLE
Accept logo parameter as string

### ISSUE THREAD

**Issue by klebershimabuku** [Irio/mymoip#17]
Actually the `logo` parameter needs to be a `Symbol`. So it is required to convert the desired parameter from `String` to `Symbol` and also it needs to be downcase.

Doing something like:

```
logo = params[:logo].downcase.to_sym
```

This could be automatically done by the gem. 

Allowing us to pass only a string:

```
logo = params[:logo]
```

@Irio what do you think?

**Comment by Irio** [comment]
Sure, accept a String it's really worthy, just don't see need for a `downcase`.

If you have the logo as `"American Express"`, your idea would produce this:

``` ruby
'American Express'.downcase.to_sym
#=> :"american express"
```

In one of mine uses:

``` ruby
# Controller
@available_cards = [
    ["American Express", "american_express"],
    ["Diners", "diners"],
    ["Hipercard", "hipercard"],
    ["Mastercard", "mastercard"],
    ["Visa", "visa"]
  ]

# View
select "credit_card", "logo", @available_cards, { prompt: "Selecione a bandeira" }, required: true
```

Maybe we could make this array a constant in MyMoip namespace, like  `MyMoip::AVAILABLE_CARDS`. If it is useful.

**Comment by klebershimabuku** [comment]
I like the idea for creating a constant for the available cards.

**Comment by Irio** [comment]
What about this?
https://github.com/Irio/mymoip/compare/2e969bf...4ae5c5b
https://gist.github.com/3949860 (I prefer not create this readable array inside MyMoip because it's really related to Rails, that will get a dedicated gem as soon this become stable enough)

**Comment by klebershimabuku** [comment]
Seems good :-)

**Comment by Irio** [comment]
I think this was a great addition, thanks for the idea. It is already available on `0.2.5`.
